app/lib/widgets/action/action_screen.dart

```diff
@@ -62,7 +62,7 @@ class _ActionScreenState extends State<ActionScreen> {
   Widget build(BuildContext context) {
     return Scaffold(
         appBar: AppBar(
-          backgroundColor: Theme.of(context).colorScheme.inversePrimary,
+          backgroundColor: Theme.of(context).colorScheme.surface,
           title: Text(AppLocalizations.of(context)!
               .actionScreenTitle(widget.patient.name, widget.action.name)),
```

Comment:
I think it would be more fitting to have something like `Maßnahme durchführen` here to be consistent with other screens like `Maßnahme erfolgreich`

Reply:
Done here: https://github.com/MANVSim/MANVSim/pull/417